### PR TITLE
:octocat: Day2(psql): query-and-update

### PR DESCRIPTION
**Pull request by mahabubx7**
### :octocat: Day2(psql): query-and-update
Added all required queries and schemas.
Also added the screenshoot below.

Screenshot 2023-06-06 062237
Screenshot 2023-06-06 062313
Screenshot 2023-06-06 062323
Screenshot 2023-06-06 062333
Screenshot 2023-06-06 062341
Screenshot 2023-06-06 062347
Screenshot 2023-06-06 062354
Screenshot 2023-06-06 062405

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -4,8 +4,9 @@ CREATE TABLE animals (
...
     name varchar(100) not null,
     date_of_birth date not null,
-    escape_attempts int,
-    neutered boolean,
-    weight_kg decimal not null
+    escape_attempts int not null,
+    neutered boolean not null,
+    weight_kg decimal not null,
+    species varchar(100) not null
 );
```
Comment (1GORDON) [inline schema.sql:11]: Great job @mahabubx7 🤝

- [x] As stated in the project requirements, I would recommend you add the command that adds the `species` column to your `animals` table. Add this to the `schema.sql` file.
- `ALTER TABLE animals ADD species varchar(255);`

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mahabubx7 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (KanzaTahreem) [inline README.md:1]: - [OPTIONAL] Please consider removing the following highlighted (placeholders that start with `>` ) ⛏🤓These instructions are used to guide you on what you can add in each respective section which helps in creating a professional README ✨
 
  image
  image
  image
  image

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @mahabubx7 🙋‍♀️

Great work implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Table is updated, perfect!
-  ✅ Great work using the right Gitflow 
-  ✅ Good work with queried

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments below_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**